src/DotNetWorker.Core/Hosting/WorkerOptions.cs

```diff
@@ -21,5 +22,11 @@ public class WorkerOptions
         /// Gets the collection of input converters.
         /// </summary>
         public InputConverterCollection InputConverters { get; } = new InputConverterCollection();
+
+        /// <summary>
+        /// Gets and sets the flag for opting in to surfacing user-code-thrown
+        /// exceptions to app insights. 
+        /// </summary>
+        public bool EnableUserCodeException { get; set; } = false;
```

Comment:
This is a good candidate for removal in 2.x, where this would always be the default behavior. Also update the app insights reference as discussed. 

@kshyju for awareness.

Reply:
Filed #1082 to track this item.